pkg/test/pods.go

```diff
@@ -96,6 +96,19 @@ func Pod(overrides ...PodOptions) *v1.Pod {
 	}
 }
 
+// Pods creates homogeneous groups of pods based on the passed in options, evenly divided by the total pods requested
+func Pods(total int, options ...PodOptions) []*v1.Pod {
+	pods := []*v1.Pod{}
+	count := 0
+	for _, opts := range options {
+		for i := 0; i < total/len(options); i++ {
+			opts.Name = fmt.Sprintf("fake-test-pod-%d", count)
```

Comment:
Were you getting conflicts with randomdata.Sillyname? Consider using that as the prefix if you need to suffix -%d